### ISSUE TITLE
[mobx ^2.5.0] mobx.observable property inside of mobx.extendObservable doesn't return an ObservableValue instance

### ISSUE THREAD

**Issue by nikolovp** [mobxjs/mobx#558]
Hello,

I have a store with a state object inside of it. I want to extend the state with some observable primitive values.
```
this.state = {};
mobx.extendObservable(this.state, {
  title: mobx.observable('hello')
});
```

With mobx v2.4.4 `this.state.title` is of type ObservableValue so I can attach some observers. But with mobx v2.5.0 `this.state.title` is not an observable value anymore.

mobx.observable(<scalar_value>) still returns an ObservableValue instance but it behaves differently if I use it inside of mobx.extendObservable.

Here is an example with mobx v.2.4.4: https://jsfiddle.net/ppetrov/emrLLhk7/

EDIT: mobx v2.5.0 example: https://jsfiddle.net/ppetrov/4wocz8y4/

**Comment by andykog** [comment]
@nikolovp, thanks for reporting. As a quick workaround you can use:
```jsx
mobx.extendObservable(this.state, {
  title: mobx.asReference(mobx.observable('hello'))
});
```
Or, obviously, that could be rewritten like this: https://jsfiddle.net/Sl1v3r/rg0oqkod/ 

@mweststrate, converting ObservableValue to a plain value is not the desired behaviour, right?

**Comment by mweststrate** [comment]
Ah this is a bug indeed I think, introduced [here](https://github.com/mobxjs/mobx/commit/d5bab8cc6d39bf492d7bf37b995edcb995f1970b#diff-147a1f3618d08f1399ce092b72872853R83) (the idea was that this is consistent with `computed`, but that is actually another use case, and getters are now the idiomatic way). I'll revert the behavior.

**Comment by mweststrate** [comment]
Fix will be part of 2.5.2
